extension/view/js/InPageTranslation.js

```diff
@@ -174,6 +176,51 @@ class InPageTranslation {
         sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated=\"\"] [x-bergamot-translated~=\"is-excluded-node\"] { border: 4px dashed red; }", 4);
     }
 
+    addQEStylesheet() {
+        const element = document.createElement("style");
+        document.head.appendChild(element);
+        if (!element.sheet) return;
+        const sheet = element.sheet;
+        sheet.insertRule(`
+        [x-bergamot-word-score].bad { background-image:
+            linear-gradient(45deg, transparent 65%, red 80%, transparent 90%),
+            linear-gradient(135deg, transparent 5%, red 15%, transparent 25%),
+            linear-gradient(135deg, transparent 45%, red 55%, transparent 65%),
+            linear-gradient(45deg, transparent 25%, red 35%, transparent 50%);
+          background-repeat:repeat-x;
+          background-size: 8px 2px;
+          background-position:0 95%;
+        }`, 0);
+        sheet.insertRule(`
+        [x-bergamot-sentence-score].bad {
+            background: rgba(255, 128, 128, 0.8);
+          }`, 1);
+        sheet.insertRule(`
+        [x-bergamot-sentence-index].highlight-sentence {
+            background: rgba(255, 255, 128, 0.8);
+          }
+        `,2);
+    }
+
+    addQualityClasses () {
+        document.querySelectorAll("[x-bergamot-sentence-score]").forEach(el => {
+            const sentenceScore = parseFloat(el.getAttribute("x-bergamot-sentence-score"));
+            el.classList.toggle("bad", sentenceScore < 0 && sentenceScore < Math.log(0.5));
```

Comment:
The check `sentenceScore < Math.log(0.5)` makes the check`sentenceScore < 0` redundant.